cl/settings/third_party/aws.py

```diff
@@ -3,15 +3,7 @@
 env = environ.FileAwareEnv()
 DEVELOPMENT = env.bool("DEVELOPMENT", default=True)
 
-
 # S3
-if DEVELOPMENT:
-    AWS_ACCESS_KEY_ID = env("AWS_DEV_ACCESS_KEY_ID", default="")
-    AWS_SECRET_ACCESS_KEY = env("AWS_DEV_SECRET_ACCESS_KEY", default="")
-else:
-    AWS_ACCESS_KEY_ID = env("AWS_ACCESS_KEY_ID", default="")
-    AWS_SECRET_ACCESS_KEY = env("AWS_SECRET_ACCESS_KEY", default="")
```

Comment:
Hm, these are used in prod. Are you sure we can remove them safely?

Reply:
I am pretty sure boto is picking them up from the environment vars.
But just to be sure I just reverted this change. Too risky.

Reply:
> Hm, these are used in prod. Are you sure we can remove them safely?

mmm that is a good question, we can probably leave it as I suggested at https://github.com/freelawproject/courtlistener/issues/4018#issue-2277950289